stop-training.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+stackName=$1
+shift
+if [[ $stackName ]]; then
+aws cloudformation delete-stack --stack-name $stackName
```

Comment:
Hi @leoclavero, thanks for the PR!

I don't think it's ready in it's current form to be merged.  What your script does is forceably terminates the CloudFormation stack, which doesn't therefore complete the graceful activity that occurs when training completes, i.e. it doesn't: -

1. Upload the completed model to the upload folder 
2. Import the completed model into the DeepRacer console

To provide this functionality as part of safe termination we need to update the PR so that it doesn't merely delete the CloudFormation stack, but that it invokes the 'Termination Lambda'.  This could be achieved in one of two ways: -

1. Modify the Event Bridge rule to a time that's 1-2 minutes in the future from the point you run the stop script, i.e. edit the below.  If you go down this route the EventBridge rule is not currently an output of the stack so would need adding for you to find it: -

![image](https://github.com/aws-deepracer-community/deepracer-on-the-spot/assets/53598199/842d9f29-e245-43d1-a155-9ec978efe5ef)

Reply:
@leoclavero I'm cancelling this PR in favour of this one which handles things gracefully - https://github.com/aws-deepracer-community/deepracer-on-the-spot/pull/111